src/api/content-rest-api/docs/TagsApi.md

```diff
@@ -447,3 +449,38 @@ parameter are returned in addition to those specified in the **fields** paramete
 
 [**TagEntry**](TagEntry.md)
 
+<a name="createTags"></a>
+## createTags
+> TagEntry[] createTags(tags: TagBody[])
+
+Create specified by **tags** list of tags.
+
+### Example
+
+```javascript
+import { AlfrescoApi, TagsApi} from '@alfresco/js-api';
+
+const alfrescoApi = new AlfrescoApi({
+    hostEcm: 'http://127.0.0.1:8080'
+});
+
+const tagsApi = new TagsApi(alfrescoApi);
+
+tagsApi.createTags(tags).then((data) => {
+  console.log('API called successfully. Returned data: ' + data);
+}, function(error) {
+  console.error(error);
```

Comment:
can we console the error here, well am not sure.

Reply:
It is only documentation file. That code is not executed. It's only example. We have similar examples for other places for example:
<img width="924" alt="image" src="https://user-images.githubusercontent.com/115619721/212320467-10118727-2ab5-4af6-a052-8fa3057af9b1.png">

Reply:
then it's fine, no issues in it.